pkg/tlsx/jarm/regex.go

```diff
@@ -0,0 +1,7 @@
+package jarm
+
+import "regexp"
+
+var (
+	JarmHashRegex = regexp.MustCompile("(?m)0{62}")
```

Comment:
why do we need this regex? this seems like unncessary

Reply:
true, checked the library code and it's already returning a sha256 hex encoded